Mathlib/Data/Matrix/RowCol.lean

```diff
@@ -136,6 +136,16 @@ theorem row_mulVec [Fintype n] [NonUnitalNonAssocSemiring α] (M : Matrix m n α
   ext
   rfl
 
+theorem row_mulVec_eq_const [Fintype m] [NonUnitalNonAssocSemiring α]  (v w : m → α) :
+    Matrix.row ι v *ᵥ w = Function.const _ (v ⬝ᵥ w) := rfl
+
+theorem mulVec_col_eq_const [Fintype m] [NonUnitalNonAssocSemiring α] (v w : m → α) :
+    v ᵥ* Matrix.col ι w = Function.const _ (v ⬝ᵥ w) := rfl
+
+theorem row_mul_col_eq [Fintype m] [Mul α] [AddCommMonoid α] (v w : m → α) :
```

Comment:
```suggestion
theorem row_mul_col_eq_const [Fintype m] [Mul α] [AddCommMonoid α] (v w : m → α) :
```
no?

Reply:
The RHS doesn't mention `Function.const`

Reply:
Then I'd suggest removing the `_eq`

Reply:
```suggestion
theorem row_mul_col [Fintype m] [Mul α] [AddCommMonoid α] (v w : m → α) :
```